Formula/alot.rb

```diff
@@ -0,0 +1,102 @@
+  end
+
+  resource "configobj" do
+    url "https://pypi.python.org/packages/source/c/configobj/configobj-5.0.6.tar.gz"
+    sha256 "a2f5650770e1c87fb335af19a9b7eb73fc05ccf22144eb68db7d00cd2bcb0902"
+  end
+
+  resource "zope.interface" do
+    url "https://pypi.python.org/packages/ea/a3/38bdc8e8bd068ea5b4d21a2d80eca1547cd8509318e8d7c875f7247abe43/zope.interface-4.2.0.tar.gz"
+    sha256 "36762743940a075283e1fb22a2ec9e8231871dace2aa00599511ddc4edf0f8f9"
+  end
+
+  resource "twisted" do
+    url "https://pypi.python.org/packages/18/85/eb7af503356e933061bf1220033c3a85bad0dbc5035dfd9a97f1e900dfcb/Twisted-16.2.0.tar.bz2"
+    sha256 "a090e8dc675e97fb20c3bb5f8114ae94169f4e29fd3b3cbede35705fd3cdbd79"
+  end
+
+  resource "urwid" do
+    url "https://pypi.python.org/packages/source/u/urwid/urwid-1.3.1.tar.gz"
+    sha256 "cfcec03e36de25a1073e2e35c2c7b0cc6969b85745715c3a025a31d9786896a1"
+  end
+
+  resource "urwidtrees" do
+    url "https://github.com/pazz/urwidtrees/archive/1.0.2.tar.gz"
+    sha256 "703f4b161b930a26a461a3e3e695f94237ac75e2a52b87613e49093d9aa76034"
+  end
+
+  resource "pygpgme" do
+    url "https://pypi.python.org/packages/source/p/pygpgme/pygpgme-0.3.tar.gz"
+    sha256 "5fd887c407015296a8fd3f4b867fe0fcca3179de97ccde90449853a3dfb802e1"
+  end
+
+  resource "python-magic" do
+    url "https://pypi.python.org/packages/source/p/python-magic/python-magic-0.4.11.tar.gz"
+    sha256 "89021e288d6efd22cde2842349d79939b9664efdbf99f5790c9862a67759ea94"
+  end
+
+  def install
+    ENV["PYTHONPATH"] = Formula["notmuch"].opt_lib/"python2.7/site-packages"
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
+    resources.each do |r|
+      r.stage do
+        system "python", *Language::Python.setup_install_args(libexec)
+      end
+    end
+    touch libexec/"lib/python2.7/site-packages/zope/__init__.py"
+
+    system "python", *Language::Python.setup_install_args(libexec)
+    (bin/"alot").write_env_script(libexec/"bin/alot", :PYTHONPATH => ENV["PYTHONPATH"])
+
+    pkgshare.install Dir["extra/*"] - %w[extra/completion]
+    zsh_completion.install "extra/completion/alot-completion.zsh" => "_alot"
+
+    if build.with? "sphinx-doc"
+      ENV["LC_ALL"] = "en_US.UTF-8"
+      ENV["SPHINXBUILD"] = Formula["sphinx-doc"].opt_bin/"sphinx-build"
+      cd "docs" do
+        system "make", "pickle"
+        system "make", "man", "html"
+        man1.install "build/man/alot.1"
+        doc.install Dir["build/html/*"]
+      end
+    end
+  end
+
+  test do
+    (testpath/".notmuch-config").write "[database]\npath=#{testpath}/Mail"
+    (testpath/"Mail").mkpath
+    system Formula["notmuch"].bin/"notmuch", "new"
+
+    begin
+      pid = fork do
+        $stdout.reopen("/dev/null")
+        $stdin.reopen("/dev/null")
+        exec "script", "-q", "/dev/null", bin/"alot", "--logfile", testpath/"out.log"
+      end
+      sleep 2
+    ensure
+      quiet_system "pkill", "-9", "-P", pid
```

Comment:
Does `Process.kill 9, pid` not work here? Apologies if someone asked at some point before.